resources/views/admin/internet/internet_access/list.blade.php

```diff
@@ -21,17 +66,23 @@
                     title: "@lang('internet.internet_access')",
                     field: "has_internet_until",
                     sorter: "datetime",
-                    //editor: 'input',
-                    headerFilter: 'input'
+                    editor: 'input'
```

Comment:
All `has_internet_until` fields are empty.
Its editor does not enforce any format on the input.
It is not self-explanatory how to use this filed, especially now, when the header for that field is meaningless.

- We should fix the column's name to something like 'end date'.
- On editing, add placeholder if no value is given, and validate on client side before submit.